lib/sparse/src/index/search_context.rs

```diff
@@ -161,9 +159,6 @@ impl<'a, 'b, T: PostingListIter> SearchContext<'a, 'b, T> {
                     // SAFETY: `id` is within `batch_start_id..=batch_last_id`
                     // Thus, `local_id` is within `0..batch_len`.
                     *unsafe { scores.get_unchecked_mut(local_id) } += element_score;
-
-                    // Measure CPU usage of indexed sparse search.
-                    cpu_counter.incr_mut();
```

Comment:
This is a hot loop indeed and removing this line gives ~4-10% percents on `search/mmap/random-50k` on my machine.